src/main/java/com/ardoq/adapter/ReferenceAdapter.java

```diff
@@ -26,12 +26,18 @@ private Gson gson() {
 
     public Reference deserialize(JsonElement jsonElement, Type type, JsonDeserializationContext jsonDeserializationContext) throws JsonParseException {
         Gson gson = gson();
+        JsonObject jsonObject = jsonElement.getAsJsonObject();
```

Comment:
move the `Gson gson = gson();` to the line above its first use

Reply:
fb580c5